docs/self-hosted/sso.md

```diff
@@ -115,9 +115,10 @@ REFRESH_TOKEN_COOKIE_SAME_SITE="None"
    - via SDK
 
      ```js
-     const sdk = new Directus('https://directus.myserver.com');
+     const sdk = new Directus('https://directus.myserver.com')
+          .with(authentication('cookie', { credentials: 'include' }));
 
-     await sdk.auth.refresh();
+     await sdk.refresh();
```

Comment:
I'd prefer including the imports as well, like we have it in other SDK examples:
```js
import { createDirectus, authentication } from '@directus/sdk';

const client = createDirectus('http://directus.example.com').with(authentication('cookie', { credentials: 'include' }));

await client.refresh();
```